pkg/ip/cidr.go

```diff
@@ -83,20 +83,6 @@ func IPToNetPrefix(ip net.IP) netip.Prefix {
 	return netip.PrefixFrom(a, a.BitLen())
 }
 
-// IPsToNetPrefixes returns all of the ips as a slice of netip.Prefix.
-//
-// See IPToNetPrefix() for how net.IP types are handled by this function.
-func IPsToNetPrefixes(ips []net.IP) []netip.Prefix {
```

Comment:
Do you think we might still need this for some of the other remaining `net.IP` transitions? In general I figured that we'd drop all of these helpers once the migration is complete.

Reply:
+1